snippets/base/models.py

```diff
@@ -577,11 +578,26 @@ class Meta:
 
 class TargetedCountry(CachingMixin, models.Model):
     code = CountryField('Geolocation Country', unique=True)
+    name = models.CharField(max_length=100)
+    priority = models.BooleanField(default=False)
 
     objects = CachingManager()
 
     def __unicode__(self):
```

Comment:
You could replace this with the `__str__` method and use the `python_2_unicode_compatible` decorator.

https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.encoding.python_2_unicode_compatible